esp/scm/ws_dfu.ecm

```diff
@@ -363,6 +372,7 @@ DFUInfoRequest
     [min_ver("1.37")] bool IncludeJsonTypeInfo(false);
     [min_ver("1.37")] bool IncludeBinTypeInfo(false);
     [min_ver("1.39")] ESPenum DFUChangeProtection Protect(0);
+    [min_ver("1.53")] ESPenum DFUChangeRestricted Restricted(0);
```

Comment:
Perhaps just change the name to show it's an action?  Like "Protect" rather than "Protected" above.  "Restrict", "ChangeRestricted", or "ChangeRestriction" would do.